### DISCUSSION TITLE
UseCase-2: Alert Interception

### DISCUSSION THREAD

**Discussion by FabrizioMoggio** [camaraproject/CallForwardingSignal#9] [Ideas]
Use Case description here: https://github.com/camaraproject/CallForwardingSignal/discussions/3#discussioncomment-8701847

**Comment by FabrizioMoggio** [comment]
this is a valid use case for both mobile and fixed access. Isn't it?

**Reply by chinaunicomyangfan** [reply]
In theory, it is, but in practical situations, mobile scenarios are more common. A typical example is that fraudsters use chat software such as WeChat to guide victims to enter code on their phones to activate call forwarding functions, thereby implementing subsequent fraud processes. It is difficult to achieve this kind of fraud under fixed access conditions

**Comment by FabrizioMoggio** [comment]
Do we need different Intents for this use case?  Now we have: https://github.com/camaraproject/CallForwardingSignal/discussions/6

Is required a new Intent for this use case?

In my opinion: no.

**Comment by FabrizioMoggio** [comment]
@chinaunicomyangfan I'm including this use case in the API Documentation. I'm wondering if "Telecommunication Frauds" is the right name for it. It seems to me a bit too generic. What about "Alert Denial"? Have you a better idea for a name a bit more specific on the use case you proposed? 

Any suggestion from anyone?

**Reply by chinaunicomyangfan** [reply]
I agree,"Telecommunication Frauds" is indeed generic, "Alert Denial" is better, or "Alert Interception".What do you think?

**Comment by FabrizioMoggio** [comment]
I'm fine with both. Unless someone has any better idea, please @chinaunicomyangfan propose your preference.

**Comment by chinaunicomyangfan** [comment]
@FabrizioMoggio I prefer "Alert Interception",if no one has better idea,I think we can use it.

**Comment by FabrizioMoggio** [comment]
The new name requires a modification in the API Documentation

**Comment by FabrizioMoggio** [comment]
@chinaunicomyangfan also for this use case it is only unconditional call FWD that is used, isn't it?

**Reply by chinaunicomyangfan** [reply]
Yes,it is only used in unconditional call FWD scene

**Comment by FabrizioMoggio** [comment]
Supported by the current release

**Comment by FabrizioMoggio** [comment]
@chinaunicomyangfan because we are moving to Alpha release, is it possible for you to provide a user story file?

According to Action Point: 2.1
The template is: https://github.com/camaraproject/CallForwardingSignal/blob/main/documentation/API_documentation/CFS_UserStory_UseCase-1.md

I know the user story it is optional for an Alpha but it would be nice to have.
